tests/testthat/test-navbar.R

```diff
@@ -208,3 +208,9 @@ test_that("can specific link target", {
     )
   })
 })
```

Comment:
Maybe tweak the test case for `"data_navbar() can re-order default elements"` to include search?